src/libsystemd-network/fuzz-dhcp-server.c

```diff
@@ -54,7 +54,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         struct in_addr address = { .s_addr = htobe32(UINT32_C(10) << 24 | UINT32_C(1))};
         _cleanup_free_ uint8_t *duped = NULL;
 
-        if (size < sizeof(DHCPMessage))
+        if (outside_size_range(size, sizeof(DHCPMessage), 600))
```

Comment:
I don't think this limit should be added here. I think the bogus options file should be removed instead

Reply:
OK, I'll remove the options file instead.